wordpress-mobile/WordPress-Android
# Feature/extract notifications

Fixes #8273 

Replaces a raw json with type safe object `FormattableContent` in the `NotificationsUtils.getSpannableContentForRanges` and related methods/classes.

To test:
1. Open the app
2. Go to Notifications
3. Make sure all types of notifications are correctly displayed
4. Open detail of each type and make sure it's correctly displayed
